scripts/lint.py

```diff
@@ -226,6 +226,7 @@ class LibRuleNotInLibDirectory(Lint):
     recommendation = "Move the rule to the `lib` subdirectory of the rules path"
 
     def check_rule(self, ctx, rule):
+        logger.debug(rule.meta)
```

Comment:
maybe this snuck in?

Reply:
yeah, added it temporarily to debug the CI runs